alerting/provider/gitlab/gitlab.go

```diff
@@ -0,0 +1,147 @@
+func (provider *AlertProvider) Send(endpoint *core.Endpoint, alert *alert.Alert, result *core.Result, resolved bool) error {
+	if len(alert.ResolveKey) == 0 {
+		alert.ResolveKey = uuid.NewString()
+	}
+
+	buffer := bytes.NewBuffer(provider.buildAlertBody(endpoint, alert, result, resolved))
+	request, err := http.NewRequest(http.MethodPost, provider.WebhookURL, buffer)
+	if err != nil {
+		return err
+	}
+	request.Header.Set("Content-Type", "application/json")
+	request.Header.Set("Authorization", fmt.Sprintf("Bearer %s", provider.AuthorizationKey))
+	response, err := client.GetHTTPClient(nil).Do(request)
+	if err != nil {
+		return err
+	}
+	defer response.Body.Close()
+	if response.StatusCode > 399 {
+		body, _ := io.ReadAll(response.Body)
+		return fmt.Errorf("call to provider alert returned status code %d: %s", response.StatusCode, string(body))
+	}
+	return err
+}
+
+type AlertBody struct {
+	Title                 string `json:"title,omitempty"`                   // The title of the alert.
+	Description           string `json:"description,omitempty"`             // A high-level summary of the problem.
+	StartTime             string `json:"start_time,omitempty"`              // The time of the alert. If none is provided, a current time is used.
+	EndTime               string `json:"end_time,omitempty"`                // The resolution time of the alert. If provided, the alert is resolved.
+	Service               string `json:"service,omitempty"`                 // The affected service.
+	MonitoringTool        string `json:"monitoring_tool,omitempty"`         // The name of the associated monitoring tool.
+	Hosts                 string `json:"hosts,omitempty"`                   // One or more hosts, as to where this incident occurred.
+	Severity              string `json:"severity,omitempty"`                // The severity of the alert. Case-insensitive. Can be one of: critical, high, medium, low, info, unknown. Defaults to critical if missing or value is not in this list.
+	Fingerprint           string `json:"fingerprint,omitempty"`             // The unique identifier of the alert. This can be used to group occurrences of the same alert.
+	GitlabEnvironmentName string `json:"gitlab_environment_name,omitempty"` // The name of the associated GitLab environment. Required to display alerts on a dashboard.
+}
+
+func (provider *AlertProvider) monitoringTool() string {
+	if len(provider.MonitoringTool) > 0 {
+		return provider.MonitoringTool
+	}
+	return "gatus"
+}
+
+func (provider *AlertProvider) service(endpoint *core.Endpoint) string {
+	if len(provider.Service) > 0 {
+		return provider.Service
+	}
+	return endpoint.DisplayName()
+}
+
+// buildAlertBody builds the body of the alert
+func (provider *AlertProvider) buildAlertBody(endpoint *core.Endpoint, alert *alert.Alert, result *core.Result, resolved bool) []byte {
+	body := AlertBody{
+		Title:                 fmt.Sprintf("🚨 (%s): %s", provider.monitoringTool(), provider.service(endpoint)),
```

Comment:
```suggestion
		Title:                 fmt.Sprintf("alert(%s): %s", provider.monitoringTool(), provider.service(endpoint)),
```
For consistency across alerting providers